devices/pine64-pinephone-braveheart/kernel/default.nix

```diff
@@ -6,21 +6,22 @@
 }:
 
 (mobile-nixos.kernel-builder {
-  version = "5.5.0";
+  version = "5.6.0";
   configfile = ./config.aarch64;
   src = fetchFromGitLab {
     owner = "pine64-org";
     repo = "linux";
-    rev = "94cf851f0f4443c771a926102dee497def319b49";
-    sha256 = "1a4ch2j8hla3xd7rv38ra6bnv14lsnj0srhlh1c8vxxvwywzg815";
+    rev = "2a6074e53b019db8a4a63cb255bc29422db3fe19";
+    sha256 = "1rjs2wvbbq69mzr6i3hjdbwsr61gflzvgx13z5hf89gcp29idmcp";
   };
   patches = [
-    ./0001-dts-pinephone-Setup-default-on-and-panic-LEDs.patch
+     ./0001-dts-pinephone-Setup-default-on-and-panic-LEDs.patch
+     ./disable-power-save.patch
```

Comment:
(nitpick) It looks like this went from a two spaces indent to a three space indent.

Reply:
done!